istioctl/pkg/validate/validate.go

```diff
@@ -79,112 +70,106 @@ func (v *validator) validateResource(un *unstructured.Unstructured) error {
 			Value: mixercrd.ToBackEndResource(un),
 		})
 	}
-	return fmt.Errorf("%s.%s validation is not supported", un.GetKind(), un.GetAPIVersion())
-}
 
-var errMissingResource = errors.New(`error: you must specify resources by --filename.
-Example resource specifications include:
-   '-f rsrc.yaml'
-   '--filename=rsrc.json'`)
+	return nil
```

Comment:
Doesn't this change behavior?

Reply:
I found it strange that istioctl invalidates k8s services. I think it should only enforce validation on config that istio owns.